app/controllers/health/my_patients_controller.rb

```diff
@@ -61,13 +61,19 @@ def patient_scope
           or(patient_source.where(nurse_care_manager_id: current_user.id))
       end
 
+      patient_ids_with_qas_in_month = population.
+        joins(:qualifying_activities).
+        merge(Health::QualifyingActivity.in_range(Date.current.beginning_of_month..Date.tomorrow)).
+        pluck(:id)
```

Comment:
This is probably fine, but it could probably be a `select` instead of `pluck`, which would result in a `not in (select`.  Until we have 5,000+ patients, it probably won't matter though.